File: Java/프로그래머스/0/181906. 접두사인지 확인하기/접두사인지 확인하기.java
class Solution {
    public int solution(String my_string, String is_prefix) {
        int answer = 0;
        String str[]=new String[my_string.length()];
        for(int i=0; i<str.length;i++){
            str[i]=""; // 모두 빈 문자열로 초기값 설정
        }
        for(int i=0; i<str.length;i++){
            for(int j=0; j<=i; j++){
                str[i]+=my_string.charAt(j);
            }
        }
        for(int i=0 ; i<str.length ; i++){
            if(is_prefix.equals(str[i])){
                answer=1;
                break;
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/181906. 접두사인지 확인하기/접두사인지 확인하기.js
function solution(my_string, is_prefix) {
    let answer = 0;
    var str = new Array(my_string.length);
    for(let i=0; i<str.length;i++){
        str[i]="";
        for(let j=0; j<=i; j++){
            str[i]+=my_string.charAt(j);
        }
    }
    for(let i=0 ; i<str.length ; i++){
        if(is_prefix === str[i]){
            answer=1;
            break;
        }
    }
    return answer;
}